cypress/integration/examples/taters_spec.js

```diff
@@ -0,0 +1,41 @@
+describe('home page', () => {
+  it('Should it should have a header', () => {
+    cy.visit('http://localhost:3000')
+    .get('h1').contains('Rotten Taters 🥔')
+  })
+
+  it('Should display movies', () => {
+    cy.visit('http://localhost:3000')
```

Comment:
I'm guessing this is before refactoring to use the actual API. You probably already caught this, but you'll need an intercept here eventually.